crates/oxc_linter/src/snapshots/no_bitwise.snap

```diff
@@ -0,0 +1,83 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_bitwise
+---
+
+  ⚠ eslint(no-bitwise): Unexpected use of "^"
+   ╭─[no_bitwise.tsx:1:1]
+ 1 │ a ^ b
+   · ─────
```

Comment:
It's best to point to the operator, but we don't have a token 🤔 Are we able to get. the source and I find it 🤔

Reply:
I am stuck on how to compute the left expression's length in nested expression. e.g. 

```js
(a | b) 
// ^ this is the expected position

(a | b) | c
    //  ^ this is the expected position
```

Reply:
I think it's impossible for a AST, let's give up 😄